packages/config/src/doc/IProfOpts.ts

```diff
@@ -9,15 +9,35 @@
 *
 */
 
+import { Logger } from "../../../logger";
+
 /**
  * Options that will affect the behavior of the ProfileInfo class.
  * They are supplied on the ProfileInfo constructor.
  */
 export interface IProfOpts {
+    /**
+     * Specifies an already initialized Imperative logger that the ProfileInfo
+     * class should use. If omitted, a new logger will be instantiated.
+     */
+    imperativeLogger?: Logger;
```

Comment:
I am certain that I added the following code to ProfileInfo.initImpUtils( ) in an attempt to avoid re-initializing logging without having to make our caller pass a logger on our constructor. Now however, I cannot find this code in any commit history.  Somehow I must have lost the code before making a commit. I know this code will initialize logging if it has not been initialized. Perhaps you can test if this code works properly in a test environment where a caller has already initialized logging? The "if" test is the only thing not currently committed to ProfileInfo-for-next
```
        // initialize logging
        if (LoggerManager.instance.isLoggerInit === false) {
            const loggingConfig = LoggingConfigurer.configureLogger(
                ImperativeConfig.instance.cliHome, ImperativeConfig.instance.loadedConfig
            );
            Logger.initLogger(loggingConfig);
        }
        this.mImpLogger = Logger.getImperativeLogger();
    }
```

Reply:
To use the comment above a second required change is to import a module:
import { LoggerManager } from "../../logger/src/LoggerManager";

Reply:
Thanks, I'll implement this and get rid of the unnecessary `imperativeLogger` property.